ui/src/app/utils/constants.tsx

```diff
@@ -87,3 +87,6 @@ export const STATE_CODE_MAPPING = {
 export const JUPYTER_FILE_EXT = '.ipynb';
 export const RMD_FILE_EXT = '.Rmd';
 export const R_SCRIPT_EXT = '.R';
+export const R_FILE_EXT = '.R';
+
+export const PROXY_PATH_SUFFIX = 'app';
```

Comment:
This change doesn't affect the Jupyter runtime proxy path, right?  We should name this accordingly, with GKE_ or similar